other/check-ingress-nginx-controller-version-and-annotation-policy/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: check-ingress-nginx-controller-version-and-annotation-policy
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: ../check-ingress-nginx-controller-version-and-annotation-policy.yaml
+  - name: step-02
+    try:
+    - patch:
+        resource:
+          apiVersion: kyverno.io/v1
+          kind: ClusterPolicy
+          metadata:
+            name: check-ingress-nginx-controller-version-and-annotation-policy
+          spec:
+            validationFailureAction: Enforce
```

Comment:
Why not set `Enforce` in the policy directly?

Reply:
Hi @realshuting, I've been using audit mode in the policy, but for testing purposes, I'm switching to enforce mode to ensure that non-compliant pods are blocked, and the test passes.